README.md

```diff
@@ -33,6 +33,16 @@ This repo contains subgraphs defined using a set of standardized schemas. These
 
 > For other contributing guidelines, please refer to [Contributing.md](./docs/Contributing.md)
 
+## Video Walkthrough Series of Subgraph Development Workflow for Messari
+
+[![Deployment.json Walkthrough](https://img.youtube.com/vi/YOUTUBE_VIDEO_ID_HERE/0.jpg)](https://youtu.be/cgCNkRmM4NM)
```

Comment:
`YOUTUBE_VIDEO_ID_HERE` is this on purpose? :D 

Other than that looking good! Thanks for all work on documenting this

Reply:
No. I fixed.